hack/image-osbs.patch

```diff
@@ -4,8 +4,8 @@ Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
 <+>UTF-8
```

Comment:
as you are not changing the image.yaml file, you dont need to update this patch, otherwise I suppose git will fail to apply

Reply:
never mind, ignore this comment.